### PR TITLE
Fix on get_change with new History ordering [5477]

### PR DESCRIPTION
**Pull request by richiware**
This should fix ros2/rmw_fastrtps#280 and also fix ros2/rmw_fastrtps#281

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/1769/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/1728/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/1740/)

File: src/cpp/rtps/history/History.cpp [hunk 1]
```diff
@@ -100,27 +100,33 @@ bool History::get_max_change(CacheChange_t** max_change)
     return false;
 }
 
-bool History::get_change(const SequenceNumber_t& seq, const GUID_t& guid,CacheChange_t** change)
+bool History::get_change(const SequenceNumber_t& seq, const GUID_t& guid, CacheChange_t** change)
```
Comment (MiguelCompany) [inline src/cpp/rtps/history/History.cpp]: Each parameter on a new line

Comment (richiware) [reply src/cpp/rtps/history/History.cpp]: Done

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/1775/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/1734/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/1746/)

**Pull request merged**